src/main/webapp/resources/js/pages/projects/redux/fieldsSlice.js

```diff
@@ -1,5 +1,9 @@
 import { createAsyncThunk, createSlice } from "@reduxjs/toolkit";
-import { getMetadataFieldsForProject } from "../../../apis/metadata/field";
+import {
+  getMetadataFieldsForProject,
+  getMetadataRestrictions,
+  patchProjectMetadataFieldRestriction,
```

Comment:
I think I mentioned this above but change `patchProjectMetadataFieldRestriction` to `updateProjectMetadataFieldRestriction`

Reply:
Fixed earlier